package.json

```diff
@@ -65,6 +66,7 @@
     "@types/react-dom": "^17.0.0",
     "@types/react-router-dom": "^5.1.7",
     "@types/react-truncate": "^2.3.3",
+    "@types/styled-components": "^5.1.9",
```

Comment:
The same as the [previous](https://github.com/bouncefinance/bounce-market/pull/75/files#r648916846) comment